omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -44,10 +44,19 @@
     # Whitelist the docs necessary to serve up error pages and friendly
     # html to non-chef clients hitting this host.
     location ~ "^/[0-9]{3,3}\.(json|html)|favicon.ico|index.html$" {
```

Comment:
[NOTE]  Note that this location block is doing all the heavy lifting of disabling the welcome page.  Even though this location block won't actually match most initial requests that would result in the welcome page, those requests eventually get here: https://github.com/chef/chef-server/blob/master/omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/dispatch.lua.erb#L49  The ngx.exec does an internal redirect to index.html which then hits the 404.